lockbox-ios/Presenter/ItemDetailPresenter.swift

```diff
@@ -47,6 +47,7 @@ class ItemDetailPresenter {
             target.itemDetailStore.itemDetailId
                 .take(1)
                 .flatMap { target.dataStore.get($0) }
+                .take(1)
                 .map { item -> [Action] in
                     var actions: [Action] = []
                     if copyableFields.contains(value) {
```

Comment:
I'm… not… sure… I… understand… how this fixes the problem. Isn't this a `RouteStore` a problem?

Reply:
The subscription wasn't getting cleaned up on returning the the app, which apparently syncs on foreground (I want to clean that up in a subsequent ticket), so the `get($0)` call was pushing again. This resulted in the openwebsite action getting dispatched a second time.

Reply:
Ok, I see; thanks. In that case:

Nit: in-code comment to this effect to explain why you have two almost consecutive `take(1)`